src/Client.php

```diff
@@ -150,12 +160,18 @@ public function update(AutotaskObjects\Entity $obj)
 
     public function bulkUpdate(array $objs)
```

Comment:
Not sure I like this change.
I'd prefer a check before executing the call as such:
```
if (count($objs) > 200) {
    throw new \Exception('You can only execute a bulk update on a max of 200 objects per request');
}
```

The reasoning behind this is to not change the returning interface of `return $this->_call('update', $updateObjs);

If anyone is using this function, you are breaking it for them by making this change, whereas throwing the exception is pushing up the logic to the person using the library.